src/components/label/label.md.scss

```diff
@@ -48,6 +48,12 @@ $label-md-margin:                     $item-md-padding-top ($item-md-padding-rig
   transition: transform 150ms ease-in-out;
 }
 
+html[dir=rtl] {
+  .label-md[floating] {
```

Comment:
When it is a single block, preferably, use `[dir="rtl"] .label-md[floating]` instead of nesting. (see [segment](https://github.com/driftyco/ionic/blob/dd0b29388e832d278d1ca7c2d4a977d771226330/src/components/segment/segment.ios.scss#L133))